### PR TITLE
add default docker containers

### PR DESCRIPTION
**Pull request by bendichter**
add default locations for docker containers

this requires that the containers are uploaded with a "latest" tag

### CODE REVIEW

**Comment by alejoe91** [comment]
`latest` tags updated to all images

File: spikeinterface/sorters/runsorter.py [hunk 1]
```diff
@@ -42,34 +66,49 @@
...
+def run_sorter(
+    sorter_name: str,
+    recording: BaseRecording,
+    output_folder: str = None,
+    remove_existing_folder: bool = True,
+    delete_output_folder: bool = False,
+    verbose: bool = False,
+    raise_error: bool = True,
+    docker_image: bool = False,
+    singularity_image: bool = False,
```
Comment (alejoe91) [inline spikeinterface/sorters/runsorter.py:78]: Shouldn't these be `Union[str, bool, None]`?

Comment (bendichter) [reply spikeinterface/sorters/runsorter.py]: @alejoe91 fixed

**Review by alejoe91 — APPROVED** [review]

**Comment by alejoe91** [comment]
@bendichter just changed the common docs because they were not showing up in the docs

**Pull request merged**